File: 第1回アルゴリズム実技検定/Python/A.py
'''
===================================================================
Project Name  : 2倍チェック
File Name     : A.cpp
Encoding      : UTF-8
Creation Date : 2022/1/1
Copyright (c) 2022 Yuma Horaguchi All rights reserved.
===================================================================
'''

def main():
	input_data = input()
	if input_data.isdigit() == True:
		print(int(input_data) * 2)
	else:
		print('error')

if __name__ == '__main__':
	main()

File: 第1回アルゴリズム実技検定/Ruby/A.rb
=begin
===================================================================
Project Name  : 2倍チェック
File Name     : A.rb
Encoding      : UTF-8
Creation Date : 2022/1/1
Copyright (c) 2022 Yuma Horaguchi All rights reserved.
===================================================================
=end

def main()
	input_data = gets()
	if /^[+-]?[0-9]+$/ =~ input_data.to_s
		puts(Integer(input_data) * 2)
	else
		puts('error')
	end
end

if __FILE__ == $0
	main()
end
